.tekton/keystone-operator-pull-request.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/raukadah/keystone-operator-konflux-integration?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    # when adding filter for directory use "directory/***".pathChanged()
+    # when adding filter for git submodule use "submodule".pathChanged()
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "konflux_main" &&
+      (".tekton/keystone-operator-pull-request.yaml".pathChanged()")
```

Comment:
Something is not right here
```
{"level":"error","ts":"2024-09-12T04:17:11.875Z","logger":"pipelinesascode","caller":"matcher/annotation_matcher.go:172","msg":"there was an error evaluating the CEL expression, skipping: failed to parse expression \"event == \\\"pull_request\\\" && target_branch == \\\"konflux_main\\\" && (\\\".tekton/keystone-operator-pull-request.yaml\\\".pathChanged()\\\")\": ERROR: <input>:1:123: Syntax error: token recognition error at: '\")'\n | event == \"pull_request\" && target_branch == \"konflux_main\" && (\".tekton/keystone-operator-pull-request.yaml\".pathChanged()\")\n | ..........................................................................................................................^\nERROR: <input>:1:125: Syntax error: missing ')' at '<EOF>'\n | event == \"pull_request\" && target_branch == \"konflux_main\" && (\".tekton/keystone-operator-pull-request.yaml\".pathChanged()\")\n 
```

Reply:
I think "(".tekton/keystone-operator-pull-request.yaml".pathChanged())", this may work.